doc/schemas/signinvoice.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "bolt11"
+  ],
```

Comment:
The bolt12 invoice born already signed iirc, so maybe here the options can be:

- Support only the bolt11 and return an error in case of bolt12;
- Support both, and return a warning in case of bolt12 (and if I'm not wrong regarding the bolt12 invoice) that tells that the invoice is already signed. This warning could be useful also in case of bolt11 is already signed?